tools/snp.sh

```diff
@@ -1223,8 +1307,11 @@ main() {
 
       # TEMPORARY until sev-snp-measure is updated to pass in TCB kernel modifier flags
       # Changes in AMDESE/linux set debug_swap on by default and affect the measurement
-      sudo modprobe -r kvm_amd
-      sudo modprobe kvm_amd debug_swap=0
+	  if [ "$SVSM" = true ]; then
+		sudo modprobe -r kvm_amd
+        sudo modprobe kvm_amd debug_swap=0
+	  fi
+     
```

Comment:
Any reason why we introduced a conditional around this? Seems odd to me...

Reply:
Simple answer is this does not need for coconut-svsm. Without this works for me. I added svsm logic, based on scripts i shared though previous PR.

Reply:
This has nothing to do with coconut SVSM. This is a kernel module adjustment to make sure that the kernel feature for DEBUG_SWAP is disabled. Regardless of being inside of a COCONUT_SVSM machine or a standard guest VM running SEV-SNP, if this kernel module is enabled, it dramatically fills the buffer for the DMESG log and can potentially cause the guest to crash. This condition needs to be removed to make sure the guest is not inhibited.